install.sh

```diff
@@ -178,25 +235,217 @@ legacy_transformations_end() {
 
     # Remove legacy registration file (we moved it to another location)
     if [ -f "$INSTALL_DIR/autoupdate/registered" ]; then
-        rm -rf "$INSTALL_DIR/autoupdate"
+        rm -rf "$INSTALL_DIR/autoupdate" || LEGACY_TRANSFORM_FAILURES="true"
+    fi
+
+    legacy_transform_registered_repos || LEGACY_TRANSFORM_FAILURES="true"
+
+    if [ "$LEGACY_TRANSFORM_FAILURES" = "true" ]; then
+        echo "! There were legacy transform errors: check stderr"
+        return 1
+    fi
+
+    return 0
+}
+
+############################################################
+# Transform all comma-delimited global \`githooks.shared\`
+#   values into multiple git config values.
+#
+# Returns:
+#   1 when failed, 0 otherwise
+############################################################
+legacy_transform_split_global_shared_entries() {
+
+    CURRENT_LIST=$(git config --global --get-all githooks.shared)
+
+    # If we have more then one of these githooks.shared values
+    # we have surely passed this install update already.
+    if [ "$(echo "$CURRENT_LIST" | wc -l)" != "1" ]; then
```

Comment:
A better way would be `echo "$CURRENT_LIST" | grep -q ','` then you are sure that it contains commas or not.

Reply:
It was more a safety bail-out check... I can remove its a bit strange yes